app/assets/src/components/views/bulk_download/choose_step.scss

```diff
@@ -166,3 +166,11 @@
     margin-top: 6px;
   }
 }
+
+.notification {
+  margin-bottom: 20px;
```

Comment:
try to use our standard spacing from app/assets/src/styles/themes/_elements.scss when possible